packages/oid4vci-demo-frontend/src/configs/banqup-account.json

```diff
@@ -146,6 +147,7 @@
   "routes": [
     {
       "id": "default",
+      "vpDefinitionId": "banqup-account",
```

Comment:
this flow does not need a vpDefinitionId

Reply:
Removed